app/controllers/responses_controller.rb

```diff
@@ -0,0 +1,53 @@
+class ResponsesController < ApplicationController
+  def index
+    @responses = Response.all
+    render json: ResponseSerializer.new(@response).serializable_hash, status: 200
+  end
+
+  def show
+    @response = Response.find(params[:id])
+
+    render json: ResponseSerializer.new(@response).serializable_hash, status: 200
+  end
+
+  def create
+    @response = Response.new(response_params)
+    @response.user_id = 1
+    @response.topic_id = 1
+
```

Comment:
`@respons.user_id` and `@resnpose.topic_id` are temporary, we should find a way to pass these from the payload
<img width="904" alt="Screen Shot 2020-11-03 at 12 25 55 PM" src="https://user-images.githubusercontent.com/14843767/98034859-a5b8cc80-1de5-11eb-8ced-3db5ba953365.png">

Or figure out what is the best practice.